presidio-analyzer/presidio_analyzer/predefined_recognizers/il_id_recognizer.py

```diff
@@ -0,0 +1,24 @@
+from presidio_analyzer import Pattern
+from presidio_analyzer import PatternRecognizer
+# pylint: disable=line-too-long,abstract-method
+
+ISRAELI_ID_REGEX = r'\b[0-9]{7,10}\b'
+
+# Todo (Task: TBD): fix context for unsupported languages
+CONTEXT = [
+    "תעודה",
+    "זהות",
+    "תעודת",
+    "ת.ז"
+]
+
+
```

Comment:
Maybe for a consecutive PR but there's validation logic for Israeli IDs

Reply:
will add.